### PR TITLE
docs: updating readme file, mostly minor formatting

### PR DESCRIPTION
**Pull request by seesmof**
fix no closing ** tag after the Laridian Book Builder link ✝️💗🙏

Hallelujah thank You Jesus Christ our Holy Lord Almighty Living GOD Most High

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,11 @@
-BibleMultiConverter
-===================
+# BibleMultiConverter
```
Comment (schierlm) [inline README.md:1]: Why?

You may ask Why Not? 2-line headlines in markdown are a lot more visible when editing the file in an editor without syntax highlighting, and the syntax is equivelent in both Markdown and RestructuredText.

**Review by schierlm — CHANGES_REQUESTED** [review]
Thank you for trying to actually fix things. But please do not change things (like the ones I commented) that are purely subjective.

Give me
- one space after ordered lists
- two-line style headings
- asterisk for formatting where possible

and I can merge your pull request. If you don't want to do it, I can do your actual change in my own commit later today.

**Comment by seesmof** [comment]
alrighty most definitely brother Mike, GOD bless ✝️💗🙏

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+  "editor.formatOnSave": false
+}
```
Comment (schierlm) [inline .vscode/settings.json:3]: Please add to .gitignore instead

**Review by schierlm — CHANGES_REQUESTED** [review]

**Comment by schierlm** [comment]
Thanks for your contribution. God bless you!

**Pull request merged**